pgscatalog_utils/download/Catalog.py

```diff
@@ -128,7 +116,7 @@ def _query_api(self, url: str):
         while retry < self._max_retries:
             try:
                 logger.info(f"Querying {url}")
-                r: requests.models.Response = requests.get(url, headers=self._user_agent)
+                r: requests.models.Response = requests.get(url, headers=config.headers())
```

Comment:
Wondering if we should have an object for:
```requests.get(url, headers=config.headers())```
as the same code is also used in `pgscatalog_utils/download/download_file.py` (line 27).
Probably overkilling.